# Archived-R-files
I have uploaded all my archived R files from over the years. 
README for Archived R Files
This is a compilation of all my R files from before my thesis project. The separate REC score file was used for our paper. Please note that these are some of my early files. They are extremely rough so I apologize in advance, but my documentation and code structure has greatly improved. I try to leave some notes here to help you navigate my old files.
graphs for methyl new.r 4/17/18: USED FOR THE METHYL project. I generated graphs using this file.
methyl regression2.r 4/3/18: USED FOR THE METHYL project. I generated the regression graphs using this file.
methyl regression.r 4/3/18: Older one from above.
add_promoters.r 12/8/17: USED FOR THE METHYL project. I generate the hg19_edit_L1HS_promoters.gtf using this file.
GDC file.r 12/4/17: Used to download files from legacy also located in server
methyl.r 11/27/17: USED FOR THE METHYL project. It generated a file for each cancer with a list of names
Graphs for Dr Han 9/27/17: create graphs
Combined Graph of best 3 mirna perâ€¦9/18/17: created graphs
REC genes fix.r 9/6/17: Another version of REC score
new patients REC scores fix.r 9/1/17: Another version of REC score
Gene correlation p and r values.r 8/8/17: Calculating the P and R values of cancer data
log 2 by cancer of new patients.r 8/4/17 Log 2 fold change from the base files done manually. Computes P and r values.
Individual Cancer log ratios 2.r 7/26/17: Contains the log2 ratios of all the cancers
Pvalue for 1M Nulls.r 7/26/17 When I was doing all those nulls for some reason
Mutation correlations. R 7/21/17: USED IN THE METHYL project. Script that I used to save under median mutations.
REC genes.r 7/21/17: Another REC score function
qval test.r 7/21/17: NEB gene Q value abstraction
Generating correlation Graph.r 7/21/17
Log2 of all new patients2.r 7/21/17 Final log
getintersect.py 7/21/17   Used for SKAT analysis. I saved it for my own records but I think it might be on 107 somewhere
Number of patients per cancer.r 7/7/17 Me calculating the numer of patients and correlations with graphs for miRNA analysis.
New patients REC scores 6/23/17: Old REC score stuff
Checking high lows.r 6/22/17: Me checking some of the miRNAs that showed to be significant.
Pvalue for 1M Nulls2.r 6/22/17: Another one of those Null generating files we never used.
Log 2 of all new patients.r 6/16/17: older fold change script
Null file histogram.r 5/23/17: Generates a histogram
Plots Paper.r 5/17/17: Me creating plots for miRNA data
Nullfilehistogram.r 5/1/17: Another file for histograms for the null files
Null of REC Score.r 5/1/17: Null file generator
PermutationFinal.r 4/25/17: File to generate a bunch of histograms used in conjunction with a bash script
All Cancer by groups.R 4/20/17: MiRNA data for cancer patients. Older file
PermutationTest.r: Another premutation script.
nullFileStrip.py 4/20/17: Python script for my null file generation
Q value for SKAT results.r 4/20/17: SKAT file
NullofRECScore.r 4/6/17: Another null file script
Nicky Transposon Match.r 3/27/17: Looks for patient matches for SKAT analysis
Compare t200.r 3/23/17: Matches with top 200
Cancer RPM.r 3/17/17: Generating the reads per million of the cancer
Median all mutations.r 3/9/17	IMPORTANT FOR SKAT
SKATCancerType.R 3/8/17: USED FOR SKAT. THIS IS THE SCRIPT TO RUN SKAT
My Box Plot Script.r 3/2/17: Box plots
Processeddata issue.r 2/24/17: R script to look at the processed data
All Mutations Currated.r 1/26/17: L1HS cancer files Used to combine all mutations. USED FOR SKAT
Mutation correlation ALL.r 1/26/17: Running correlations 
Hknot based on individual microRNA 9/7/16: Another REC Score function
All Cancer pvalues log.R 8/17/16: MiRNA analysis
All cancer log ratios 4/25/16
All Cancer pvalues L1HS.R 3/28/16: old file for L1HS cancer
File REC score analysis: Contains all my scripts for the final REC Score calculations for the paper
File miRNA mutation by cancer type: Contains all the miRNA results for the paper
File Gene correlation my cancer typeL Contains all the gene results for the paper
